README.md

```diff
@@ -32,6 +32,37 @@ The repository has the following permanent branches:
 
  * **apk** This branch contains the apks for the code in the develop branch. The apks are automatically updated when a commit is pushed to `develop` branch.
 
+
+##Run Server Locally To Test Android
+
+ * **Run server on Local machine methods.**
+	1. Run by local ip :-
+		 Connect your mobile and laptop on the same network. Type command ipconfig if you are using Linux or ipconfig if you are using windows. Modify the run.py by following line 
+
+		if __name__ == "__main__":
+		application.debug=True
+		application.run(host='<local-ip>',port=5000)
+	        <local-ip> be like 192.168.xx.xx
+	2. By using ngrok :-
+	        Install ngrok on your system by following command:- 
+		brew cask install ngrok
+		ngrok http <port>
+		    This outputs, among some other information, a line like
+	        Forwarding                    http://4cc5ac02.ngrok.io -> localhost:5000
+	        Now, you can navigate to http://4cc5ac02.ngrok.io on any device that is connected to the Internet, and this URL redirects to localhost:5000 of your laptop.
+
+ 
+ * **Connectivity with Android App.**
```

Comment:
Could you include code blocks inside markdown for code blocks? Like this:
```
code block
```
I am saying this for the piece of code you have there.

Reply:
done.